projects.md

```diff
@@ -0,0 +1,10 @@
+# Projects
+
+Existing projects in Go that can be used as examples.
+
+## HelloFresh
+
+- https://github.com/hellofresh/remote-config-service
```

Comment:
Just thinking here, maybe we could have a private https://github.com/avelino/awesome-hellofresh pretty similar to https://github.com/avelino/awesome-go

So in order to put projects in there we should review the code, suggest some improvements and  etc... all via Pull Requests, WDYT?